nuxeo-features/rest-api/nuxeo-rest-api-test/src/main/java/org/nuxeo/ecm/restapi/test/TestOAuth2Provider.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Gabriel Barata <gbarata@nuxeo.com>
+ */
+package org.nuxeo.ecm.restapi.test;
+
+import org.nuxeo.ecm.platform.oauth2.providers.NuxeoOAuth2ServiceProvider;
+
+/**
+ * @since 8.4
+ */
+public class TestOAuth2Provider extends NuxeoOAuth2ServiceProvider {
```

Comment:
It will be removed. It was being used previously to the last refactoring. Thanks for pointing it out.